apps/dolly-backend/src/main/resources/db/migration/V1.7.2__BestillingProgressNom.sql

```diff
@@ -0,0 +1,6 @@
+------------------------------
+-- A L T E R   T A B L E S  --
+------------------------------
+
+alter table bestilling_progress
+add column nom_status text;
```

Comment:
text tror jeg gir en clob som vi ikke trenger her. Hva med varchar(500) i stedet?

Reply:
Aha, takk for tipset 😄